test/utils/update/fetch.managing.officers.private.data.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { describe, expect, test, beforeEach, jest } from '@jest/globals';
+import { fetchManagingOfficersPrivateData } from "../../../src/utils/update/fetch.managing.officers.private.data";
+import { getManagingOfficersPrivateData } from "../../../src/service/private.overseas.entity.details";
+import { isActiveFeature } from "../../../src/utils/feature.flag";
+import { logger } from "../../../src/utils/logger";
+import { ApplicationData } from "../../../src/model";
+import { MOCK_MANAGING_OFFICERS_PRIVATE_DATA, MOCK_APP_DATA_MOS } from "../../../test/__mocks__/session.mock";
+
+jest.mock("../../../src/utils/feature.flag");
+jest.mock("../../../src/service/private.overseas.entity.details");
+jest.mock("../../../src/utils/logger");
+
+const mockIsActiveFeature = isActiveFeature as jest.Mock;
+const mockGetManagingOfficersPrivateData = getManagingOfficersPrivateData as jest.Mock;
+const mockLoggerInfo = logger.info as jest.Mock;
+const mockLoggerError = logger.errorRequest as jest.Mock;
+
+describe("fetchManagingOfficersPrivateData", () => {
+  let appData: ApplicationData, req: Request;
+
+  beforeEach(() => {
+    appData = MOCK_APP_DATA_MOS;
+    jest.clearAllMocks();
+  });
+
+  test("should map Managing Officers Private Data when data is available", async () => {
+    mockIsActiveFeature.mockReturnValue(false);
+    mockGetManagingOfficersPrivateData.mockResolvedValue(MOCK_MANAGING_OFFICERS_PRIVATE_DATA);
+
+    await fetchManagingOfficersPrivateData(appData, req);
+
+    expect(appData.update?.review_managing_officers_individual?.length).toEqual(1);
```

Comment:
We could also added the equivalent check for corporate MOs:

`expect(appData.update?.review_managing_officers_corporate?.length).toEqual(1);`